WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2468,7 +2468,23 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
 
         if (resultCode != Activity.RESULT_OK) {
-            return;
+            switch (requestCode) {
+                case RequestCodes.MULTI_SELECT_MEDIA_PICKER:
+                case RequestCodes.SINGLE_SELECT_MEDIA_PICKER:
+                case RequestCodes.PHOTO_PICKER:
+                case RequestCodes.STOCK_MEDIA_PICKER_SINGLE_SELECT:
+                case RequestCodes.MEDIA_LIBRARY:
+                case RequestCodes.PICTURE_LIBRARY:
+                case RequestCodes.TAKE_PHOTO:
+                case RequestCodes.VIDEO_LIBRARY:
+                case RequestCodes.TAKE_VIDEO:
+                case RequestCodes.STOCK_MEDIA_PICKER_MULTI_SELECT:
+                    mEditorFragment.mediaSelectionCancelled();
+                    return;
+                default:
```

Comment:
NitPick: We might want to consider removing `default` and listing all the "cases" so when someone adds a new item into the ENUM, lint will inform them that they need to handle the new case in this switch. Wdyt?

Reply:
I tried this, but it seems the lint is not working in this way. Perhaps this is because the `RequestCode` is an enum-like-class with static int fields rather than an actual enum? I've left a comment above the switch, which will hopefully help.

Reply:
My bad - I didn't realize `RequestCode` is technical a regular int. Sorry for the misleading suggestion. Thank you for the comment ❤️